specifyweb/frontend/js_src/lib/components/Reports/index.tsx

```diff
@@ -160,7 +176,6 @@ export function ReportsView({
 
 function ReportRow({
   resources: unsortedResources,
-  icon,
```

Comment:
The argument was removed from here, but should have also been removed from the TypeScript section:
```ts
readonly icon: JSX.Element;
```